man/systemd-boot.xml

```diff
@@ -214,6 +214,16 @@
       </varlistentry>
     </variablelist>
 
+    <para>The boot menu is shown when a non-zero menu time-out has been configured. If the menu time-out has
```

Comment:
This should be "timeout", not "time-out". We predominantly use the first spelling, and the dictionary agrees:

timeout 

     A period of time after which an error condition is raised if
     some event has not occured. ...

time-out

      a brief suspension of play;